README.rst

```diff
@@ -36,7 +36,7 @@ Language    Status  Parser
 ==========  ======  ==========================================================
 Python      Stable  Custom using `astroid <https://github.com/PyCQA/astroid>`_
 Go          Alpha   `godocjson <https://github.com/readthedocs/godocjson>`_
-Javascript  Alpha   `jsdoc <http://usejsdoc.org/>`_
+Javascript  Alpha   `jsdoc <https://jsdoc.app/>`_
```

Comment:
Couldn't find usejsdoc.org, but the <https://github.com/jsdoc/jsdoc> project uses the link above :+1: